src/components/navbar-header-panels/app-menu-panel.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ExtensionSlot } from "@openmrs/esm-framework";
+import { HeaderPanel } from "carbon-components-react/es/components/UIShell";
+
+interface AppMenuProps {
+  expanded: boolean;
+}
+
+const AppMenuPanel: React.FC<AppMenuProps> = ({ expanded }) => {
+  return (
+    <HeaderPanel
+      aria-label="App Menu Panel"
+      style={{
+        height: "12rem",
```

Comment:
I know its set right to 12rem, but I wonder why. Are the designs like this? Why not make it consistent with the other side menu? (user)

Reply:
made it consistent with the other side menu.